R/check_forecasts.R

```diff
@@ -210,6 +195,26 @@ check_forecasts <- function(data) {
 }
 
```

Comment:
Is there really not a function in `checkmate` to do this?

Reply:
🥲

Reply:
You mean one a function for what exactly? 
Also noting that in the next PR I replaced `check_forecasts()` by something called `validate()`. The main reason is that it felt aesthetically displeasing to have all those checkmate functions with their clear naming convention alongside `check_forecasts()`...